engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -239,12 +239,36 @@ object WorkflowDescriptor {
   val CallLogsDirOptionKey = "call_logs_dir"
   val OptionKeys: Set[String] = Set(WorkflowLogDirOptionKey, WorkflowOutputsOptionKey, CallLogsDirOptionKey)
 
+  // This is, frankly, an aberration. But don't worry - it's going away soon :D
+  // TODO: KILL IT WITH FIRE!
+  def getBackendFromOptions(optionsString: String): Backend = {
+    val backendString = WorkflowOptions.fromJsonString(optionsString) flatMap { x => x.get("backend")}
+    backendString match {
+      case Success(x) => CromwellBackend.backend(x)
```

Comment:
why not roll this into the `flatMap` above and follow that with `getOrElse` to return the default backend?